charged-ieee/lib.typ

```diff
@@ -52,14 +52,15 @@
 
   // Adapt supplement in caption independently from supplement used for
   // references.
+  set figure.caption(separator: [.])
   show figure: fig => {
     let prefix = (
       if fig.kind == table [TABLE]
       else if fig.kind == image [Fig.]
       else [#fig.supplement]
     )
     let numbers = numbering(fig.numbering, ..fig.counter.at(fig.location()))
-    show figure.caption: it => [#prefix~#numbers: #it.body]
+    show figure.caption: it => [#prefix~#numbers#figure.caption.separator #it.body]
```

Comment:
Allow overriding per caption as well, not only with set rules.

```suggestion
    show figure.caption: it => [#prefix~#numbers#it.separator #it.body]
```

Reply:
To match with the default show rule's behaviour, the trailing space should be part of the separator rather than the show rule.

Reply:
Thanks, I've updated my suggestion